### DISCUSSION TITLE
Why cupy is so faster than torch as multi-step neuron backend?

### DISCUSSION THREAD

**Discussion by realamirhe** [fangwei123456/SpikingFlow#133] [Q&A] [answered]
Is the torch used as a CPU only for computation or there is another reason where cupy and torch performance gap is this sharp?

**Comment by fangwei123456** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/38352109/138548528-24493cff-646b-4f9a-9624-0b7d33553e9f.png)

**Reply by realamirhe** [reply]
Thank you for your response 🙏🏻

**Reply by realamirhe** [reply]
Is there a way you can provide some resources about optimization information you've sent?

**Reply by fangwei123456** [reply]
https://pytorch.org/tutorials/recipes/recipes/tuning_guide.html#fuse-pointwise-operations

Hi, I have not met a speicifc reference that discusses the fused oprtation.
